core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -286,4 +286,17 @@
       font-size: 2vw;
     }
   }
+
+  @media only screen and (max-width: 420px) {
+    .dropdown-label {
+      font-size: 18px;
+      margin-right: 2vw;
+      width: 100px !important;
```

Comment:
Why is `!important` needed I think we try not to use it.

Reply:
This is needed to make sure the alignment of the labels and dropdowns across two components is consistent on widths < 420px. Removing `!important` from here will cause issues with the widths.

Reply:
Okay, then please add an explanatory comment explaining (precisely) why is this needed.

Reply:
Done